production/docker/docker-compose.yaml

```diff
@@ -42,7 +42,7 @@ services:
   prometheus:
     image: prom/prometheus:v2.27.0
     ports:
-      - 9090:9090
+      - 9090
```

Comment:
I have monitoring on my local system, so mapping to the default ports on localhost conflicts with that. This syntax will map container port `9090` to a random port on the host, so it's still accessible without conflicting